cli/src/main.rs

```diff
@@ -23,26 +64,41 @@ pub async fn main() {
       .arg_from_usage("<APP_ID> 'Specifies the alan app to terminate'")
     )
     .subcommand(SubCommand::with_name("upgrade")
-      .about("Deploys an .agz file to an existing app in the cloud provider described in the deploy config at ~/.alan/deploy.json")
+      .about("Deploys your repository to an existing app in the cloud provider described in the deploy config at ~/.alan/deploy.json")
       .arg_from_usage("<APP_ID> 'Specifies the alan app to upgrade'")
-      .arg_from_usage("<AGZ_FILE> 'Specifies the .agz file to deploy'")
     );
 
   let matches = app.get_matches();
   match matches.subcommand() {
     ("new",  Some(matches)) => {
-      let agz_file = matches.value_of("AGZ_FILE").unwrap();
-      let cloud_alias = matches.value_of("CLOUD_ALIAS").unwrap();
-      new(agz_file, cloud_alias).await;
+      let config = get_config();
+      let cloud_alias = matches.value_of("CLOUD_ALIAS").unwrap_or(
+        config.keys().take(1).next().unwrap()
+      );
+      let body = json!({
```

Comment:
we are doing this outside the `deploy` module so that the alan cli can figure out what to pass in a simpler way, right?

Reply:
Yes. It's a bit duplicative between the two, but rather than writing some weird abstraction to handle potentially unset args, this seemed simpler since we need different code in both places, anyways.